fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/api/resource/conditions/v1/package-info.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Provides a way of conditionally loading JSON-based resources. This can be used with
+ * recipes, loot tables, advancements, predicates, and item modifiers. Conditions are
```

Comment:
This is only by default for fabric, but mods are free to use conditions too for whatever they want. I use them in Modern Dynamics for the datapack upgrade specifications. :wink:

Reply:
(I am suggesting a clarification)